templates/devspace.yaml.tpl

```diff
@@ -160,6 +160,8 @@ dev:
         value: Always
       - op: remove
         path: spec.containers[0].securityContext
+      - op: remove
```

Comment:
This seems useful, but _technically_ `resourcer` is already doing this today, since it mutates the pod before it is created.

Reply:
Ah. I did not consider that. We can close this then.